week-3/implement/get-angle-type.js

```diff
@@ -21,3 +21,17 @@
 // Identify Reflex Angles:
 // When the angle is greater than 180 degrees and less than 360 degrees,
 // Then the function should return "Reflex angle"
+
+function getAngleType(angle) {
```

Comment:
360 is not taken care of

Reply:
thank for you your feedback I will  fix it